youtube_dl/extractor/nonktube.py

```diff
@@ -0,0 +1,68 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+import re
+
+
+class NonkTubeIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?nonktube\.com\/video/(?P<id>[0-9]+)/[0-9a-zA-Z\-]*'
+    _TESTS = [{
+        'url': 'https://www.nonktube.com/video/118636/sensual-wife-uncensored-fucked-in-hairy-pussy-and-facialized',
+        'md5': 'ccd2df2aef9f44e51773c8ea59de6fd1',
+        'info_dict': {
+            'id': '118636',
+            'ext': 'mp4',
+            'title': 'Sensual Wife Uncensored Fucked In Hairy Pussy And Facialized',
+            'thumbnail': 'https://www.nonktube.com/media/videos/tmb_2/118636/default.jpg',
+            'uploader': 'https://www.nonktube.com/user/cumdrinkingwife',
+            'age_limit': 18
+        }
+    }, {
+        'url': 'https://www.nonktube.com/video/118698/2-japanese-girls-sharing-one-dick-uncesnored',
+        'md5': 'bb4bdd0a61f591e2f5f7429dc860327e',
+        'info_dict': {
+            'id': '118698',
+            'ext': 'mp4',
+            'title': '2 Japanese Girls Sharing One Dick Uncesnored',
+            'thumbnail': 'https://www.nonktube.com/media/videos/tmb_2/118698/default.jpg',
+            'uploader': 'https://www.nonktube.com/user/cumdrinkingwife',
+            'age_limit': 18
+        }
+    }]
+    DOWNLOAD_URL_TEMPLATE = 'https://cdn.nonktube.com/%s.mp4'
```

Comment:
Ok. I checked out the extractor for lovehomeporn which seems to be similar

```python
 info = self._extract_nuevo(
            'http://lovehomeporn.com/media/nuevo/config.php?key=%s' % video_id,
            video_id)
```

I found a similar url in the website - https://www.nonktube.com/media/nuevo/config.php?key=118636. However, unlike the former which showed the xml when I loaded the link in the browser, here it shows **"restricted area"** I tried passing the original video URL to extract_nuevo function which also didn't work. In both cases, it raised error.